### PR TITLE
Added diverse and affinity group IDs to flow history update dump

### PR DESCRIPTION
**Pull request by nrydanov**
Closes: #3807

### CODE REVIEW

File: src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/SkipPathsAndResourcesDeallocationAction.java [hunk 1]
```diff
@@ -59,8 +60,4 @@ public void perform(State from, State to, Event event, FlowUpdateContext context
             stateMachine.fire(Event.UPDATE_ENDPOINT_RULES_ONLY);
         }
     }
-
-    private FlowPath getFlowPath(PathId pathId) {
```
Comment (dpoltavets) [inline src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/SkipPathsAndResourcesDeallocationAction.java:63]: This method, which does not throw exceptions if the path is not found, is written here for a reason. Do not change logic without a good reason.

**Review by dpoltavets — CHANGES_REQUESTED** [review]

File: src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/SkipPathsAndResourcesDeallocationAction.java (hunk 1 above)
Comment (nrydanov) [reply src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/SkipPathsAndResourcesDeallocationAction.java:63]: Didn't notice, thanks

**Review by dpoltavets — APPROVED** [review]

File: src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/CompleteFlowPathRemovalAction.java [hunk 2]
```diff
@@ -40,11 +40,12 @@ public CompleteFlowPathRemovalAction(PersistenceManager persistenceManager) {
...
         Flow originalFlow = RequestedFlowMapper.INSTANCE.toFlow(stateMachine.getOriginalFlow());
+        originalFlow.setAffinityGroupId(stateMachine.getOriginalAffinityFlowGroup());
+        originalFlow.setDiverseGroupId(stateMachine.getOriginalDiverseFlowGroup());
+
         removeOldPrimaryFlowPaths(originalFlow, stateMachine);
         removeOldProtectedFlowPaths(originalFlow, stateMachine);
-
-        Flow flow = getFlow(stateMachine.getFlowId());
-        removeRejectedFlowPaths(flow, stateMachine);
+        removeRejectedFlowPaths(originalFlow, stateMachine);
```
Comment (niksv) [inline src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/CompleteFlowPathRemovalAction.java:48]: I this there is a bug here. Rejected Paths are new paths. And original flow doesn't contains them. That is why we had to get flow from DB with new paths. We can discuss it on some call

**Review by niksv — CHANGES_REQUESTED** [review]

File: src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/CompleteFlowPathRemovalAction.java (hunk 2 above)
Comment (nrydanov) [reply src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/CompleteFlowPathRemovalAction.java:48]: Ok

**Pull request merged**